spock-spring/spring.gradle

```diff
@@ -12,20 +12,20 @@ if (springVersion.startsWith("2.")) {
 }
 
 dependencies {
-  compile project(":spock-core")
-  compile "org.springframework:spring-test:$springVersion", provided
-  compile "org.springframework:spring-beans:$springVersion", provided
-  compile "org.springframework:spring-context:$springVersion", provided
-
-  // not used directly at compile-time, but needed by groovyc
-  testCompile "org.springframework:spring-core:$springVersion"
-  testCompile "org.springframework:spring-jdbc:$springVersion"
-  testCompile "org.springframework:spring-tx:$springVersion"
-  testCompile "javax.inject:javax.inject:1"
-  testCompile libs.groovySql // for groovy.sql.Sql
-
-  testRuntime libs.h2database
-  testRuntime libs.log4j
+  api project(":spock-core")
+  implementation "org.springframework:spring-test:$springVersion", provided
```

Comment:
Should this be in API scope since it's expected to use Spring's annotations?

Reply:
The normal usage is for users to depend their `springVersion`, I hope they don't use the ancient version we use here. But I'm not so sure if it should be in `api` or not.

Reply:
In that case `implementation` makes sense to me.